ST_params.c

```diff
@@ -787,11 +787,11 @@ static void _rgroup_init( void) {
         break;
      }
      x=sscanf( inbuf, "%s %f %f %d %f %d %d %d %d %d %f %d %d %d %f %d %d %f %f"
-               " %f %f %d %f %f", name, &space, &density, &estab, &slow, 
+               " %f %f %d %f %f %f", name, &space, &density, &estab, &slow,
                &stretch, &xres, &estann, &turnon, &styr, &xgro, &veg_prod_type,
                &killyr, &killfreq_startyr, &killfreq, &extirp, &mort, 
                &prop_killed, &prop_recovered,&grazing_frq, &prop_grazing,
-               &grazingfreq_startyr, &biomass, &transpiration);
+               &grazingfreq_startyr, &biomass, &transpiration, &live_biomass);
      if (x < 22) {
```

Comment:
The user error in `_rgroup_init()`, `if (x < 22) ...`, hasn't been updated in a while with new rgroup inputs. With the current change, I think it should be `if (x  < 25) ...` if an appropriate error for "Too few columns in groups" should be issued.

Reply:
@dschlaep good catch thanks!